inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/ClusterController.java

```diff
@@ -58,6 +59,37 @@
         return Response.success(clusterInfoService.list(request));
     }
 
+    @RequestMapping(value = "/common/save", method = RequestMethod.GET)
```

Comment:
what does the `common` stand for? it's better to use an explicit name.

Reply:
**I will modify this problem in the next issue and pull request.**